MULTICOIN-INFO.md

```diff
@@ -0,0 +1,69 @@
+# Multi-coin compatibility
+
+This wallet is not limited to one icon, but is compatible with most coins based on the Skycoin technology. For a
+coin to be compatible, it must have a public node updated to at least version 0.24 and, ideally, it's REST api
+should have not been modified (if the api is modified, the wallet could fail depending on whether any of the
+required endpoints were modified).
+
+## Adding a coin
+
+To add a coin, you must Add the required data about the coin. Most data must be added in a new class, in a new
```

Comment:
I would replace Add with provide just to avoid repetition:
To add a new coin, you must provide the required data about the coin.